### PR TITLE
[Independent-Publisher] Added icanhazdadjoke.com connector

### PR DESCRIPTION
**Pull request by Laskewitz**
Added the icanhazdadjoke.com connector

---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verfied by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/icanhazdadjoke/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "current_page": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "current_page",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/icanhazdadjoke/apiDefinition.swagger.json:89]: @Laskewitz Can you please update the description to `Current page`

File: independent-publisher-connectors/icanhazdadjoke/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "limit",
+                  "title": "Limit"
+                },
+                "next_page": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "next_page",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/icanhazdadjoke/apiDefinition.swagger.json:101]: @Laskewitz Can you please update the description to `Next page`.

Can you please update all the similar description values of properties with valid ones in the swagger.

File: independent-publisher-connectors/icanhazdadjoke/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Daniel Laskewitz"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/icanhazdadjoke/apiProperties.json:6]: @Laskewitz Can you please add `stackOwner` property as well..

Documentation - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@Laskewitz I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by Laskewitz** [comment]
@sriyen-msft  Fixed! 👍🏼

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
Deployment Pending..

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/icanhazdadjoke/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# icanhazdadjoke ([icanhazdadjoke.com](https://icanhazdadjoke.com))
+
+[icanhazdadjoke.com](https://icanhazdadjoke.com) is the largest selection of dad jokes on the internet. Now supporting many different integrations to ensure you can access the dad jokes that you need wherever you are.
+
```
[the remark this answers is not in the document]
Comment (Laskewitz) [reply independent-publisher-connectors/icanhazdadjoke/README.md:4]: @sriyen-msft readme updated.

**Comment by sriyen-msft** [comment]
@Laskewitz  Update - We are preparing your connector for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes between 7-10 business days. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@Laskewitz  Deployment freeze notice: Due to Thanksgiving week in the United States, we have temporarily halted the deployments of all the connectors till Tuesday, 30th November 2021.

We will begin the deployments on Wednesday, 1st December 2021. If your connector is queued up for Wednesday (1st December 2021) deployment schedule, the production deployment of your connector in all regions will be expected to be completed by Thursday, 9th December 2021. Thanks

**Comment by sriyen-msft** [comment]
@Laskewitz Apologies for the delay in response.

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program. Thanks

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**